packages/console/src/pages/ConnectorDetails/components/ConnectorTabs/index.tsx

```diff
@@ -19,28 +19,38 @@ const ConnectorTabs = ({ target, connectorId }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { data } = useSWR<ConnectorDTO[]>(`/api/connectors?target=${target}`);
 
-  if (!data) {
+  const connectors = useMemo(() => {
+    if (!data) {
+      return [];
+    }
+
+    return data.filter(({ enabled }) => enabled);
+  }, [data]);
+
+  if (connectors.length === 0) {
+    return null;
+  }
+
+  if (connectors.length === 1 && connectors[0]?.platform === ConnectorPlatform.Universal) {
```

Comment:
Is it possible that the connectors.length equals 1 and the platform is native?

Reply:
yes, try wechat